### ISSUE TITLE
[Feature request] An option to view next up options immediately after playback has stopped

### ISSUE THREAD

**Issue by mddfe** [Tikipeter/repository.tikipeter#73]
First of all, thanks again for all of the time you've spent making Fen the best add-on out there.

I'll start by saying that I love the next up window that pops up. It looks great and really helps to make the viewing experience feel premium, like we're using a nice subscription service. There is one big difference between Fen's next up and something like Netflix, and that's knowing when it is appropriate to show the information about the next episode. For the time being, it's impractical for something like Fen to determine the ideal time to show this information.

I personally hate if a dialogue pops up and distracts (or possibly spoils) the final moments of a dramatic TV episode. Even with Netflix, I feel like sometimes they don't give particularly dramatic episodes time to 'breath' and are too quick to start throwing you into another episode. On the other hand, it's annoying to have to wait (or fast-forward a slight amount) to force the dialogue to come up if the episode has a particularly long credit sequence. In the past you were kind enough to implement an option to show the dialogue based on the amount of time remaining, rather than a %. That technique seemed a lot more logical to me, but I understand that other people preferred the original method and having too many options can cause things to get messy.

So what I suggest is adding an option to prompt the user to watch the next episode or not once playback is stopped. Once enough of the video has been viewed to be considered 'watched' (and marked as such on Trakt), scraping for the next episode should start. If playback is stopped after the video has been flagged as 'watched' (either from the user stopping the video manually, or the video completely finishing) the user will be given the option to view the next episode or not. This way, the timing is up to the user for each individual episode and it only requires one additional user input. It still saves the user the time of having to refresh their widgets/next up page and scrape for the next episode.

Maybe you've already considered this, and Kodi simply give you many options to do things when playback is stopped. Or maybe it's just not a good idea for reasons I've not considered. I feel bad that in the past you implemented my suggestion to base next up on time rather than %, since other people didn't seem to care for it.

Anyways, just a thought. Thanks again for everything you have done.

TLDR: an option to present the user with next-up dialogue immediately after an episode has stopped playing

**Comment by Tikipeter** [comment]
Set the notification for early and use "Notification" for the setting "Next Episode Alert Method". That's as close as you'll get.
